### DISCUSSION TITLE
ultimate-cube 还是 UltimateCube

### DISCUSSION THREAD

**Discussion by SpecterShell** [G3G4X5X6/ultimate-cube#27] [General]
之前把软件投到了 WinGet，刚刚发现软件名称变了。
软件内的名称全部更换成小写形式的 `ultimate-cube` 了，那大写形式的 `UltimateCube` 可以用吗？

**Comment by G3G4X5X6** [comment]
仓库名字应该稳定了，但是程序打包的名字不太稳定哈，目前还是稳定的
